src/app/dataset-view/additional-components/dataset-settings-component/tabs/webhooks/create-subscription-modal/create-subscription-modal.model.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Kamu Data, Inc. and contributors. All rights reserved.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the LICENSE file.
+ */
+
+import { WebhookSubscriptionInput, WebhookSubscriptionStatus } from "src/app/api/kamu.graphql.interface";
+export interface SubscribedEventType {
+    name: string;
+    value: string;
+}
+
+export interface CreateWebhookSubscriptionSucces {
+    input: WebhookSubscriptionInput;
+    subscriptionId: string;
+    status?: WebhookSubscriptionStatus;
```

Comment:
When status is undefined? Webhook always has status.

Reply:
The result does not contain a status when created.

![image](https://github.com/user-attachments/assets/46429fc1-cf1c-4221-a9a6-471717513a96)